platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -218,7 +219,7 @@ public SwirldsPlatform(@NonNull final PlatformComponentBuilder builder) {
         notificationEngine = blocks.notificationEngine();
 
         currentAddressBook = initialState.getAddressBook();
-        currentRoster = initialState.getRoster();
+        currentRoster = AddressBookUtils.createRoster(currentAddressBook);
```

Comment:
Minor: I actually want to remove this method. We have the `RosterRetreiver.buildRoster()` that we use in most places. But that can be done later if you prefer to use this one for whatever reason.